zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/Leader.java

```diff
@@ -1716,6 +1720,7 @@ public void revalidateSession(QuorumPacket qp, LearnerHandler learnerHandler) th
         DataInputStream dis = new DataInputStream(bis);
         long id = dis.readLong();
         int to = dis.readInt();
+        dis.close();
```

Comment:
Would this be better as:
```suggestion
        long id;
        int to;
        try (DataInputStream dis = new DataInputStream(bis)) {
            id = dis.readLong();
            to = dis.readInt();
        }
```
The `bis` variable could also be included in the same try-with-resources block (but GitHub review interface won't let me select that line to include it in the suggested changes).

Reply:
Yep. Corrected in new commit.